evil-surround.el

```diff
@@ -65,8 +65,11 @@ Each item is of the form (TRIGGER . (LEFT . RIGHT)), all strings.
 Alternatively, a function can be put in place of (LEFT . RIGHT).
 This only affects inserting pairs, not deleting or changing them."
   :group 'surround
-  :type '(repeat (cons (regexp :tag "Key")
-                       (symbol :tag "Surround pair"))))
+  :type '(alist
```

Comment:
@wbolster thanks for the pointer, and I apologize in advance because I'm really an elisp beginner. I read the section but still, I did not find any `alist` function. 

Shouldn't the `alist` in the `(alist ...` you wrote be a function? If so, why isn't this function listed in the gnu doc section you've linked? I see all sorts of functions like `assoc`, `assq` but no `alist`. What am I missing here? Thanks